admin_guide/allocating_node_resources.adoc

```diff
@@ -109,68 +102,72 @@ $ yum install libcgroup-tools
 $ cgget -g memory  /system.slice | grep memory.usage_in_bytes
 ----
 +
-If this value is close to your `system-reserved` mark, you can increase the `system-reserved` value. 
+If this value is close to your `system-reserved` mark, you can increase the `system-reserved` value.
 
-* Use the {product-title} xref:../scaling_performance/using_cluster_loader.html#scaling-performance-using-cluster-loader[Cluster Loader]
+* Use the {product-title} xref:../scaling_performance/using_cluster_loader.html#scaling-performance-using-cluster-loader[cluster loader]
 to measure performance metrics of your deployment at various cluster states.
 
 [[computing-allocated-resources]]
-== Computing Allocated Resources
+== Computing allocated resources
 
 An allocated amount of a resource is computed based on the following formula:
 
 ----
-[Allocatable] = [Node Capacity] - [kube-reserved] - [system-reserved] - [Hard-Eviction-Thresholds]
+[Allocatable] = [Node Capacity] - [system-reserved] - [Hard-Eviction-Thresholds]
 ----
 
 [NOTE]
 ====
-The withholding of `Hard-Eviction-Thresholds` from allocatable is a change in behavior to improve
-system reliability now that allocatable is enforced for end-user pods at the node level.
-The `*experimental-allocatable-ignore-eviction*` setting is available to preserve legacy behavior,
-but it will be deprecated in a future release.
+The withholding of `Hard-Eviction-Thresholds` from allocatable improves system reliability because the value for allocatable is enforced for pods at the node level. The `*experimental-allocatable-ignore-eviction*` setting is available to preserve legacy behavior, but it will be deprecated in a future release.
```

Comment:
s/`*experimental-allocatable-ignore-eviction*`/`experimental-allocatable-ignore-eviction`
(unless there's regularity of this type of formatting already throughout 3.11)